other-docs/guides/upgrading/v14.md

```diff
@@ -182,5 +182,70 @@ If you are using the premium version of the plugin, follow
 the [instructions on the Yoast website](https://yoast.com/help/how-to-install-yoast-plugins-using-composer/)
 .
 
+#### Lazy loading via GaussHolder functionality
+
+As WordPress provides lazy loading of images by default Altis no longer provides
+that functionality. See
+the [Lazy Loading announcement](https://make.wordpress.org/core/2020/07/14/lazy-loading-images-in-5-5/)
+for more details.
+
+If you still require the Gaussholder image placeholder functionality, you can
+add [humanmade/Gaussholder](https://github.com/humanmade/Gaussholder) to your project.
+
+```bash
+## Add Gaussholder library
+composer require "humanmade/gaussholder"
+```
+
+Then hook into the `gaussholder.image_sizes` filter.
+
+```php
+add_filter( 'gaussholder.image_sizes', function ( $sizes ) {
+	$sizes['medium'] = 16;
+	$sizes['large'] = 32;
+	$sizes['full'] = 84;
+	return $sizes;
+} );
+```
+
+Note: You can experiment to determine the best sizes (blur radius) to use. See
+the [documentation on Github](https://github.com/humanmade/Gaussholder#readme).
+
 ## Headline Features
 
+
+### WordPress 6.1
+
+WordPress 6.1 brings with it further refinement of the site building experience. Site creation is now more intuitive.
+Several writing-focused improvements will delight your content creators. A new block-based theme 'Twenty Twenty-Three'
+now comes as standard. Additionally, there is improved accessibility, and performance.
+
+The [WordPress 6.1 Field Guide](https://make.wordpress.org/core/2022/10/12/wordpress-6-1-field-guide/) is a great place
+to learn about what's new. Here are some of the highlights:
+
+- Twenty Twenty-Three: A fresh default theme
```

Comment:
I don't think we're shipping the theme though, perhaps it's not worth mentioning it?

Reply:
Oh OK. 
Strangely we have twenty twenty-two available on a new build! 
![Screenshot 2023-02-14 at 12 36 53](https://user-images.githubusercontent.com/358499/218740858-abc1f9e1-297f-4526-88f4-def6f4b5c18e.png)